blockchain.go

```diff
@@ -422,17 +432,26 @@ func bootstrapLedger(
 	return nil
 }
 
-func configureBootstrapProcedure(ctx fvm.Context, flowAccountKey flowgo.AccountPublicKey, supply cadence.UFix64) *fvm.BootstrapProcedure {
-	if ctx.LimitAccountStorage {
-		return fvm.Bootstrap(flowAccountKey,
+func configureBootstrapProcedure(conf config, flowAccountKey flowgo.AccountPublicKey, supply cadence.UFix64) *fvm.BootstrapProcedure {
+	options := make([]fvm.BootstrapProcedureOption, 0)
+	options = append(options,
+		fvm.WithInitialTokenSupply(supply),
+	)
+	if conf.StorageLimitEnabled {
+		options = append(options,
 			fvm.WithInitialTokenSupply(supply),
```

Comment:
Already added above (https://github.com/onflow/flow-emulator/pull/28/files#diff-1fb2891d150a66e1319495e4f10a7ce04855476836a3a6b17f253e312f37d23bR438)

```suggestion
```

Reply:
whoops, missed that one!